spring-integration-core/src/main/java/org/springframework/integration/dsl/IntegrationFlowBeanPostProcessor.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.integration.config.dsl;
```

Comment:
I don't think it's too strict to move classes in this package to some other place.

May just fix `@see` JavaDocs in those classes who uses this one?

Reply:
Sorry; I don't know what you mean; S101 doesn't look at javadocs, just bytecode.

The **package** tangle between `config` and `dsl` is because this class references lots of classes in `dsl` and lots of `...Spec` classes in `dsl` reference classes in `config` (e.g. `CEFB`, `SPCAFB`). So the simplest fix was to move this to the dsl. Happy to do something else if you have a suggestion.

Reply:
Ah! I see now. That's not a simple class-to-class relationship. That's how those in `dsl` are consuming classes in this `config` package. And later opposite the `IntegrationFlowBeanPostProcessor` uses classes from the `dsl` package. 

Good to know. There is always something to study every day :smile: 
Thank you!

So, never mind. I'm fine with your current solution on the matter.